src/test/java/com/radixdlt/serialization/SerializeObjectEngine.java

```diff
@@ -0,0 +1,60 @@
+package com.radixdlt.serialization;
+
+import org.junit.Test;
+
+import java.util.function.Supplier;
+
+import static com.radixdlt.serialization.SerializationTestUtilsEngine.testEncodeDecode;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assume.assumeFalse;
+
+/**
+ * Raft of tests for serialization of objects.
+ * <p>
+ * Note that the tests that round-trip types through the serializer
+ * are not run for {@link Polymorphic} types, as these types do not
+ * serialize to themselves, but to one of their superclasses.
+ *
+ * @param <T> The type under test.
+ */
+public abstract class SerializeObjectEngine<T> {
```

Comment:
Uh, please, fix this! Copy paste of `SerializeObject` basically.... bah!

Reply:
Yeap.  There is still some duplicate stuff hanging around from where the engine was split from core.